snippets/example-besluit-met-geannoteerde-bijlage.html

```diff
@@ -0,0 +1,41 @@
+<div prefix="eli: http://data.europa.eu/eli/ontology# foaf: http://xmlns.com/foaf/0.1/ besluit: http://data.vlaanderen.be/ns/besluit# prov: http://www.w3.org/ns/prov#" resource="http://data.lblod.info/id/behandeling-van-agendapunten/82b89f04-7398-4c12-890c-a130decac4f8" typeof="besluit:BehandelingVanAgendapunt">
+<div property="prov:generated" resource="http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8" typeof="besluit:Besluit">
+  <h4 class="h4" property="eli:title" datatype="xsd:string">Goedkeuring gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren.</h4>
+  <span property="eli:language" resource="http://publications.europa.eu/resource/authority/language/NLD" typeof="skos:Concept">NL</span>
+  <p property="eli:description" datatype="xsd:string">De gemeenteraad besliste met algemene stemmen om het nieuw subsidiereglement omtrent het gebruik van herbruikbare luiers en toebehoren voor kinderen van 0 tot 3 jaar, zoals opgenomen in bijlage bij dit besluit, goed te keuren en in werking te laten treden vanaf 1 mei 2019.</p>
+
+  <div property="besluit:motivering" lang="nl">
+    <h5>Juridische context</h5>
+    <ul class="bullet-list">
+      <li>Het decreet<a property="eli:cites" href="https://codex.vlaanderen.be/doc/document/1029017" typeof="http://data.europa.eu/eli/ontology#LegalExpression" >over het lokaal bestuur van 22 december 2017</a>, meer bepaald artikels 40 en 41, betreffende de bevoegdheden van de gemeenteraad.</li>
+    </ul>
+    <br>
+  </div>
+  <h5>Beslissing</h5>
+  <div property="prov:value" datatype="xsd:string">
+    <div property="eli:has_part" resource="http://data.lblod.info/artikels/f3366b82-3751-4333-8531-b9b58b6ccb7f" typeof="besluit:Artikel">
+      <div property="eli:number" datatype="xsd:string">Artikel 1</div>
+      <span style="display:none;" property="eli:language" resource="http://publications.europa.eu/resource/authority/language/NLD" typeof="skos:Concept">NL</span>
+      <div property="prov:value" datatype="xsd:string">
+        om het nieuwe subsidiereglement omtrent het gebruik van herbruikbare luiers en toebehoren voor kinderen van 0 tot 3 jaar, zoals opgenomen in bijlage bij dit besluit, goed te keuren en in werking te laten treden vanaf 1 mei 2021.
+      </div>
+    </div>
+  </div>
+  <h5>Bijlagen</h5>
+  <ul>
+    <li><span resource="http://data.lblod.info/id/besluiten/82b89f09-7398-4c12-890c-a130decac4f8" property="eli:related_to" rev="dct:isPartOf">Gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren.</span></li>
```

Comment:
This translates with an rdfa validator to:
```
http://data.lblod.info/id/besluiten/82b89f09-7398-4c12-890c-a130decac4f8> a besluit:Besluit ;
    ns1:isPartOf <http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8> ;
    eli:has_part <http://data.lblod.info/artikels/g3366b82-3751-4333-8531-b9b58b6ccb8d> ;
    eli:title "Gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren."^^xsd:string ;
    prov:value """
    
      Artikel 1
      NL
      
        Doel Wegwerpluiers maken een groot deel van de afvalberg uit. In het kader van afvalpreventie, hergebruik en recyclage zijn herbruikbare luiers een goed alternatief voor wegwerpluiers. Het College van Burgemeester en Schepenenvan de gemeente kan -binnen de perken van de jaarlijks op de begroting voorziene en goedgekeurde kredieten-een subsidie toekennen aan mensen die voor hun  kinderen  van  0  tot  3  jaar  herbruikbare  luiers  gebruiken.  Hiermee  wil  de  gemeente  mensen stimuleren  om  op  die  manier  een  bijdrage  te leveren  aan  het  verminderen  van  de  hoeveelheid huishoudelijk restafval.
      

    
  """^^xsd:string .

<https://codex.vlaanderen.be/doc/document/1029017> a eli:LegalExpression .

<http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8> a besluit:Besluit ;
    eli:cites <https://codex.vlaanderen.be/doc/document/1029017> ;
    eli:description "De gemeenteraad besliste met algemene stemmen om het nieuw subsidiereglement omtrent het gebruik van herbruikbare luiers en toebehoren voor kinderen van 0 tot 3 jaar, zoals opgenomen in bijlage bij dit besluit, goed te keuren en in werking te laten treden vanaf 1 mei 2019."^^xsd:string ;
    eli:has_part <http://data.lblod.info/artikels/f3366b82-3751-4333-8531-b9b58b6ccb7f> ;
    eli:language <http://publications.europa.eu/resource/authority/language/NLD> ;
    eli:related_to "Gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren." ;
    eli:title "Goedkeuring gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren."^^xsd:string ;
    besluit:motivering """
    Juridische context
    
      Het decreetover het lokaal bestuur van 22 december 2017, meer bepaald artikels 40 en 41, betreffende de bevoegdheden van de gemeenteraad.
    
    
  """@nl ;
    prov:value """
    
      Artikel 1
      NL
      
        om het nieuwe subsidiereglement omtrent het gebruik van herbruikbare luiers en toebehoren voor kinderen van 0 tot 3 jaar, zoals opgenomen in bijlage bij dit besluit, goed te keuren en in werking te laten treden vanaf 1 mei 2021.
      
    
  """^^xsd:string .
```

Shouldnt:
`  eli:related_to "Gemeentelijk subsidiereglement voor het gebruik van herbruikbare luiers en toebehoren." ;` have a uri as range?

and 
```
# the general 
<http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8>  ns1:isPartOf <http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8> . # reglement
```
Shouldn't this not be the inverse?

Reply:
this should be resolved now 
```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .

<http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8>
   rdf:type <http://data.vlaanderen.be/ns/besluit#Besluit>;

   <http://data.europa.eu/eli/ontology#has_part> <http://data.lblod.info/artikels/f3366b82-3751-4333-8531-b9b58b6ccb7f>;
   <http://data.europa.eu/eli/ontology#related_to> <http://data.lblod.info/id/besluiten/82b89f09-7398-4c12-890c-a130decac4f8> .
<http://data.lblod.info/id/besluiten/82b89f09-7398-4c12-890c-a130decac4f8>
   rdf:type <http://data.vlaanderen.be/ns/besluit#Besluit>;
   <dct:isPartOf> <http://data.lblod.info/id/besluiten/72b89f05-7398-4c12-890c-a130decac4f8>;
...
```